CharlieBackend.Api/Startup.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 using Serilog;
-using EasyNetQ;
```

Comment:
this lib not used in Startup?

Reply:
Lost it while fixes.
Returned in new comment